include/operations/OpType.hpp

```diff
@@ -48,7 +48,13 @@ namespace qc {
         Barrier,
         Teleportation,
         // Classically-controlled Operation
-        ClassicControlled
+        ClassicControlled,
+        // Noise operations
+        ATrue,
+        AFalse,
+        multiATrue,
+        multiAFalse,
+        opCount
```

Comment:
With a small exception, every member of this Enum is capitalized. So should these
```suggestion
        MultiATrue,
        MultiAFalse,
        OpCount
```